common/types/layer.go

```diff
@@ -284,10 +285,7 @@ func (l *Layer) BallotIDs() []BallotID {
 
 // Hash returns the 32-byte sha256 sum of the block IDs in this layer, sorted in lexicographic order.
 func (l Layer) Hash() Hash32 {
```

Comment:
at some point i was tempted to simply delete this method. it's too confusing with `mesh.GetLayerHash`